ocfweb/static/scss/pages/_tv.scss

```diff
@@ -4,7 +4,14 @@ $tv-hours-small: 5;
 
 .page-tv {
```

Comment:
Does it work to add these attributes to the `body` rule instead?

Reply:
You mean for the TV's base.html `body`? It does IIRC but .page-tv is on a container div immediately inside body which IMHO is slightly cleaner.